example/reduce/src/reduce-main.cpp

```diff
@@ -1,3 +1,12 @@
+/* Copyright 2019-2021 Hauke Mewes, Simeon Ehrig
```

Comment:
Just for the future: I don't think we need a range of years here. Simply changing to  the current year whenever a file is touched should be enough.